brave/brave-core
# Import payments info from chrome

Resolves https://github.com/brave/brave-browser/issues/8329

![image](https://user-images.githubusercontent.com/6786187/82414827-99002080-9ab2-11ea-9147-dd0f172d0d56.png)


## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
Test 1.
1. Launch chrome and check there is no payments method
2. Launch brave and load import dialog and check payment methods option is hidden

Test 2.
1. Launch chrome and check it has payments method
2. Launch brave and load import dialog and check payment methods option is visible
3. Start import and check chrome's payment methods are imported

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
